README.md

```diff
@@ -31,11 +31,13 @@ Versions & Dependencies
 ------------
 This project tries to follow Elasticsearch in terms of [End of Life](https://www.elastic.co/support/eol) and maintenance since 5.x
 
-| Elastica                                                                                | ElasticSearch | elasticsearch-php | PHP      |
-| --------------------------------------------------------------------------------------- | ------------- | ----------------- | -------- |
-| [7.x](https://github.com/ruflin/Elastica/tree/master)                                   | 7.x           | ^7.0              | ^7.2     |
-| [6.x](https://github.com/ruflin/Elastica/tree/6.x)                                      | 6.x           | ^6.0              | ^7.0     |
-| [5.x](https://github.com/ruflin/Elastica/tree/5.x) (unmaintained)                       | 5.x           | ^5.0              | \>=5.6   |
-| [3.2.3](https://github.com/ruflin/Elastica/tree/3.2.3) (unmaintained)                   | 2.4.0         | no                | \>=5.4   |
-| [2.x](https://github.com/ruflin/Elastica/tree/2.x) (unmaintained)                       | 1.7.2         | no                | \>=5.3.3 |
+| Elastica                                                                                | ElasticSearch | elasticsearch-php | PHP          |
+| --------------------------------------------------------------------------------------- | ------------- | ----------------- | ------------ |
+| [7.x](https://github.com/ruflin/Elastica/tree/master)                                   | 7.x           | ^7.0              | ^7.2 || ^8.0 |
+| [7.1](https://github.com/ruflin/Elastica/tree/7.1)                                      | 7.x           | ^7.0              | ^7.2 || ^8.0 |
```

Comment:
Do we need the 7.1 line as it would also be contained in 7.x?

Reply:
Technically not. I'm not strongly minded either way, it's a little verbose maybe, but it's not wrong. I guess with the distinction/split of 7.0 below, it should make sense that the one above it is "other 7.x versions" (including 7.1)

I'll remove the 7.1 line